dev-libs/intel-neo/intel-neo-21.31.20514.ebuild

```
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_PN="compute-runtime"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Intel Graphics Compute Runtime for L0 and OpenCL, for Broadwell and beyond"
+HOMEPAGE="https://github.com/intel/compute-runtime"
+SRC_URI="https://github.com/intel/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="l0 vaapi"
+
+BDEPEND="virtual/pkgconfig"
+COMMON=">=virtual/opencl-3
+	>=dev-util/intel-graphics-compiler-1.0.8173
+	>=media-libs/gmmlib-21.2.1
+	l0? ( >=dev-libs/level-zero-1.4.1 )
+	vaapi? (
+		x11-libs/libdrm[video_cards_intel]
+		x11-libs/libva
+	)"
+DEPEND="${COMMON}
+	media-libs/mesa"	# for Khronos OpenGL headers
+RDEPEND="${COMMON}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-21.31.20514-no_Werror.patch
+)
+
+DOCS=(
+	README.md
+	FAQ.md
+)
+
+S="${WORKDIR}"/${MY_P}
+
+src_configure() {
+	local mycmakeargs=(
+		-DKHRONOS_GL_HEADERS_DIR="${EPREFIX}/usr/include"
```

Nit:
```suggestion
		-DKHRONOS_GL_HEADERS_DIR="${ESYSROOT}/usr/include"
```

(because this is where DEPEND-class dependencies are [satisfied](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-720008.1))

---

Done